content/deploy/choose_cloud.md

```diff
@@ -0,0 +1,70 @@
+run your JupyterHub. These companies basically own really large computers, and what
+you do is purchase some resources on those computers to run whatever you'd like (in this
+case, a JupyterHub). However, there are often a few different kinds of offerings that
+cloud providers provide, with various pros and cons.
+
+> **An aside on using free hosted services vs. creating your own deployments**. Many cloud
+  providers offer a "free" and customized notebook service for basic usage. While these are
+  appealing on their surface, they provide a number of challenges down the road, such as
+  getting your students used to the vendor-specific interface rather than an open-source version.
+  The Jupyter Project (and the Data 8 approach) aims to maximize your choice and flexibility in
+  how and where you deploy Jupyter environments on shared infrastructure, and for this reason
+  we do not recommend using most fully-managed "free" services.
+
+With that in mind, here are a few of the most-common options for choosing your cloud resources.
+
+* **You purchase computational resources and set up yourself**. The recommmened approach. In this case, you ask the
+  cloud provider for some resources, and they give you a blank slate that you can do anything
+  with. This is the most-common approach to deploying JupyterHub (along with
+  following this guide to do the actual set-up).
+
+  * **Pros**: You have flexibility in customizing your JupyterHub exactly as you wish. You also
+    have the ability to easily switch to other cloud providers because the setup process will
+    be very similar for each.
+  * **Cons**: There is more manual setup and maintenance for you to perform. Because the resources
+    are a blank slate, you'll need to take the steps to set everything up.
+* **You use a "push-button" deployment to deploy specific resources**. Another possibility is that
+  the cloud provider has given you a single-button option to automatically deploy
+  some resources on their cloud. For example, some providers let you quickly deploy a JupyterHub
+  that you then customize. Others let you deploy an entirely customized distribution of JupyterHub
+  that theoretically "works out of the box".
+
+  * **Pros**: You can get to a running JupyterHub more quickly.
+  * **Cons**: You have less flexibility over deploying and customizing this JupyterHub. In addition,
+    you can less-easily move your infrastructure to a different cloud provider because you are
+    used to a vendor-specific workflow.
+* **You use a JupyterHub-like hosted service of a cloud provider**. The final option is that you
+  use a fully-managed service of a cloud provider, such as the Google Collaboratory, or Microsoft
+  Azure Notebooks. These often look and behave like Jupyter Notebooks, though they are often
+  built on some proprietary technology from these companies. This should be your least-preferable
```

Comment:
The language here is potentially quite off-putting, "This should be your ..." is essentially imposing your value-system on instructors, and potentially comes across as judgemental if their priority system doesn't align with yours. I would recommend that we re-work the language to be a bit more matter-of-fact, and let the pros / cons do the work of providing a structure for forming an opinion.